### PR TITLE
fix definition list syntax

### PR DESCRIPTION
**Pull request by nslater**

### CODE REVIEW

File: docs/sqlalchemy.txt [hunk 1]
```diff
@@ -309,17 +309,26 @@ fields.
...
-:param match_type (optional): The match type determines how the query_term
- is applied and the _score is created
- See `Match Types <https://crate.io/docs/stable/sql/fulltext.html?highlight=fulltext#match-types>`_
-:param options (optional): The match options further distinguish the way the
- matching process using a certain match type works.
- See `Match Options <https://crate.io/docs/stable/sql/fulltext.html?highlight=fulltext#options>`_
+:param column:
+  A reference to an index column or an existing column that is of type
+  string and is indexed. It's also allowed to pass multiple columns and
+  boosts as dict
```
Comment (christianbader) [inline docs/sqlalchemy.txt]: dot

Comment (nslater) [reply docs/sqlalchemy.txt]: what do you mean?

Comment (chaudum) [reply docs/sqlalchemy.txt]: missing full stop at end of sentence.

Comment (nslater) [reply docs/sqlalchemy.txt]: I am not correcting gramma here because it tends to mess up my prs. this is strictly a formatting change only

**Pull request merged**